src/gt4py/ir/nodes.py

```diff
@@ -397,32 +397,66 @@ class FieldRef(Ref):
 
 @enum.unique
 class NativeFunction(enum.Enum):
-    ABS = 1
-    MOD = 2
+    ABS = "abs"
+    MIN = "min"
+    MAX = "max"
+    MOD = "mod"
+
+    SIN = "sin"
+    COS = "cos"
+    TAN = "tan"
+    ARCSIN = "arcsin"
+    ARCCOS = "arccos"
+    ARCTAN = "arctan"
+
+    SQRT = "sqrt"
+    EXP = "exp"
+    LOG = "log"
+
+    ISFINITE = "isfinite"
+    ISINF = "isinf"
+    ISNAN = "isnan"
+    FLOOR = "floor"
+    CEIL = "ceil"
+    TRUNC = "trunc"
 
-    SIN = 11
-    COS = 12
-    TAN = 13
-    ARCSIN = 14
-    ARCCOS = 15
-    ARCTAN = 16
-
-    SQRT = 21
-    EXP = 22
-    LOG = 23
-
-    ISFINITE = 101
-    ISINF = 102
-    ISNAN = 103
-    FLOOR = 111
-    CEIL = 112
-    TRUNC = 113
+    @property
+    def arity(self):
+        return type(self).IR_OP_TO_NUM_ARGS[self]
+
+    @staticmethod
+    def values():
+        return [func.value for func in NativeFunction]
```

Comment:
Why is this a `staticmethod`? I think it would make more sense to implement it with as `classmethod`:

```suggestion
    @classmethod
    def values(cls):
        return [func.value for func in cls]
```

Reply:
`staticmethod` was my suggestion, but I agree this is better.